test/integration/backward_compatible/parallel/serialization/DefaultSerializersLiveTest.js

```diff
@@ -664,3 +658,50 @@ describe('DefaultSerializersLiveTest', function () {
         }
     });
 });
+
+describe('DefaultSerializersLiveTest Arrays', function () {
```

Comment:
Love the idea of separating all tests about arrays in a different block.

Reply:
What about moving some more tests here? Namely:

- buffer on client
- should deserialize Java Array
- should deserialize empty Java Array

Reply:
Fixed